src/components/Person/Person.scss

```diff
@@ -0,0 +1,16 @@
+.Person {
```

Comment:
For class name better use lowercase

Reply:
I can't, 'cause tests try to catch these classes
![image](https://github.com/user-attachments/assets/58668905-cacc-4c1f-9ad7-5241df22a0d1)